Bottom: aa26245ced259979a04db6eb937f8a8354875d3e
Top:    c2feeb2f69a4ec22963cf6300286277e51855b0a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:58:49 +0000

Remaining (bug?) from another patch..

NB: Original function had this bug before we moved it


---

diff --git a/src/misc.c b/src/misc.c
index 1729bbd..7e9f635 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -397,7 +397,7 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
   hx = p1->X + (p2->X - p1->X) / 2;
   hy = p1->Y + (p2->Y - p1->Y) / 2;
 
-  p_to_h_dist = hypot (p2->X - p1->X, p2->Y - p1->Y) / 2.;
+  p_to_h_dist = hypot (p2->X - p1->Y, p2->Y - p1->Y) / 2.;
   c_to_h_dist = p_to_h_dist / tan (TO_RADIANS (included_angle) / 2.);
 
   unit_hcx = (double)-(hy - p1->Y) / p_to_h_dist;
